core/src/main/resources/hudson/PluginManager/updates.jelly

```diff
@@ -55,183 +56,178 @@ THE SOFTWARE.
+                                </li>
+                              </j:forEach>
+                            </ul>
+                          </div>
+                        </j:if>
+                        <j:if test="${p.isNeededDependenciesForNewerJenkins(cache)}">
+                          <div class="alert alert-danger">
+                            ${%depCoreWarning(p.getNeededDependenciesRequiredCore().toString())}
+                          </div>
+                        </j:if>
+                        <j:if test="${p.hasWarnings()}">
+                          <div class="alert alert-danger">${%securityWarning}
+                            <ul>
+                              <j:forEach var="warning" items="${p.getWarnings()}">
+                                <li>
+                                  <a href="${warning.url}" rel="noopener noreferrer"
+                                     target="_blank">${warning.message}
+                                  </a>
+                                </li>
+                              </j:forEach>
+                            </ul>
+                          </div>
+                        </j:if>
+                        <j:if test="${p.isDeprecated()}">
+                          <div class="alert alert-warning">
+                            ${%deprecationWarning(p.getDeprecation().url)}
+                          </div>
+                        </j:if>
+                        <j:if test="${it.hasAdoptThisPluginLabel(p)}">
+                          <div class="alert alert-warning">
+                            ${%adoptThisPlugin}
+                          </div>
+                        </j:if>
+                      </td>
+                      <td data="${p.releaseTimestamp.time}" class="jenkins-table__cell--no-wrap">
+                        <j:if test="${p.releaseTimestamp != null}">
+                          <time datetime="${h.iso8601DateTime(p.releaseTimestamp)}"
+                                tooltip="${h.localDate(p.releaseTimestamp)}">
+                            ${%ago(h.getTimeSpanString(p.releaseTimestamp))}
+                          </time>
+                        </j:if>
+                      </td>
+                      <td>
+                        <j:choose>
+                          <j:when test="${p.installed.active}">
+                            <st:out value="${p.installed.version}"/>
+                          </j:when>
+                          <j:otherwise>
+                            <span title="${%Inactive}">
+                              (<st:out value="${p.installed.version}"/>)
+                            </span>
+                          </j:otherwise>
+                        </j:choose>
+                      </td>
+                    </tr>
+                  </j:forEach>
+
+            </table>
+          </j:when>
+          <j:otherwise>
+            <l:notice icon="symbol-up-to-date" text="${%No updates}" />
```

Comment:
```suggestion
            <l:notice icon="symbol-up-to-date" text="${%No updates available}" />
```
Given the updated presence, this sounds more fitting to me, thoughts?

Reply:
I like it!